cg/services/delivery_message/messages/utils.py

```diff
@@ -16,3 +16,7 @@ def get_scout_link(case: Case) -> str:
 def get_pangolin_delivery_path(case: Case) -> str:
     customer_id: str = case.customer.internal_id
     return f"/home/{customer_id}/inbox/wwLab_automatisk_hamtning"
+
+
+def get_statina_batch_id(case: Case) -> str:
+    return case.name.split("-")[1]
```

Comment:
Just a heads up, I don't know if that makes a difference for anything on the user end:

From the case.name for fluffy/nipt/statina cases you get this: `181213-2400953_NIPT`

This function would return: 2400953_NIPT

In the mongodb for statina, batch ids look like this: 2400953

Reply:
Good catch, we will check with prod and see what they want in the delivery message